content-src/experiments/voice-fill.yaml

```diff
@@ -0,0 +1,82 @@
+id: 14
+title: 'Voice Fill'
+slug: voice-fill
+platforms: ['addon']
+thumbnail: /static/images/experiments/voice-fill/experiments_experiment/thumbnail.png
+description: >
+  Speech recognition comes to Firefox. Voice fill lets you speak to your favorite search engines.
+introduction: >
+  <p>Voice Fill uses artificial intelligence to interpret speech input. As a participant in the Voice Fill experiment, your voice searches will teach our AI to provide smarter results.</p>
+  <p>We've added Voice Fill to Google, Yahoo and DuckDuckGo search pages, and will expand to more sites as the experiment progresses.</p>
+
+image_twitter: /static/images/experiments/voice-fill/social/mv-twitter.jpg
+image_facebook: /static/images/experiments/voice-fill/social/mv-facebook.jpg
+video_link: https://www.youtube.com/embed/lDv68xYHFXM
+changelog_url: 'https://github.com/mozilla/speaktome/blob/master/CHANGELOG.md'
+contribute_url: 'https://github.com/mozilla/speaktome/'
+bug_report_url: 'https://github.com/mozilla/speaktome/issues'
+discourse_url: 'https://discourse.mozilla-community.org/c/test-pilot/voice-fill'
+privacy_notice_url: 'https://github.com/mozilla/speaktome/blob/master/docs/metrics.md'
+measurements:
+  - >
+    When you choose to use Voice Fill, your recordings will be sent to our servers for processing to provide you with suggested text.
+  - >
+    Voice Fill will collect technical information related to the performance of the feature, such as how long it takes for our servers to process a submitted recording and whether you selected or modified any of the suggestions we gave.
+  - >
+    Voice Fill will also collect information about your use of the feature, such as how frequently Voice Fill is used and whether you used Voice Fill with specific tabs or pages such as Google, DuckDuckGo, or Yahoo.
+  - >
+    To help us improve the accuracy of our suggestions, we may retain a copy of submitted audio recordings, and related suggestion information. Audio recordings kept for this purpose will not be associated with any other personal identifiers.
+xpi_url: 'https://testpilot.firefox.com/files/@voicefill/latest'
```

Comment:
This failed for prod (obviously), and also for https://testpilot.dev.mozaws.net/files/@voicefill/latest